src/apify_helpers.js

```diff
@@ -37,25 +46,31 @@ const getValuesFromKeyValueStore = async (z, storeId, keys) => {
         return z
             .request(`${APIFY_API_ENDPOINTS.keyValueStores}/${storeId}/records/${key}`)
             .then((response) => {
+                if (response.status === 404) {
+                    values[key] = {
+                        error: `Cannot find ${key} in key-value store`,
+                    };
+                    return;
+                }
                 try {
                     const maybeObject = JSON.parse(response.content);
```

Comment:
Shouldn't we support also text types or possibly other types (images, ...)?

Reply:
I think this will be for next version, I created issue of it. #8